contracts/extbridge/ext_bridge.sol

```diff
@@ -21,19 +21,17 @@ contract ExtBridge is Bridge {
         address _tokenAddress,
         uint256 _value,
         uint64 _requestNonce,
-        uint64 _requestBlockNumber
-        //uint256 [] _extraData     // TODO-Klaytn-Servicechain : This will be applied in next PR
+        uint64 _requestBlockNumber,
+        uint256 [] _extraData
     )
         public
     {
-        uint256 offerPrice = 1; //_extraData[0];
+        uint256 offerPrice = _extraData[0];
```

Comment:
What happen if _extraData is nil?

Reply:
It makes revert. I think adding length check code is needed.